docs/notifications.md

```diff
@@ -24,6 +24,10 @@ Currently supported events:
  * new/removed P2P notary request (if `P2PSigExtensions` are enabled)
 
    Contents: P2P notary request. Filters: request sender and main tx signer.
+ * mempool_event
```

Comment:
new/removed memory pool transaction (if `MempoolSubscriptionsEnabled` is enabled)